cfgov/cfgov/tests/test_urls.py

```diff
@@ -0,0 +1,71 @@
+try:
+    from django.urls import RegexURLPattern, RegexURLResolver
+except ImportError:
+    from django.core.urlresolvers import RegexURLPattern, RegexURLResolver
+
+from django.test import TestCase, override_settings
+
+from cfgov import urls
+
+# Whitelist is a list of *strings* that match the beginning of a regex string.
+# For example, ''^admin' will match any urlpattern regex that starts with
+# '^admin'.
+ADMIN_URL_WHITELIST = [
+    '^django-admin/',
+    '^admin/',
+    '^picard/',
+    '^login',
+    '^logout',
+    '^tasks/',
+    '^csp-report/',
+    '^password/',
+    '^d/admin/',
+]
+
+
+# Based on django_extensions's show_urls command.
+def get_urlpatterns(urlpatterns, base=''):
+    urls = []
+    for p in urlpatterns:
+        if isinstance(p, RegexURLPattern) or hasattr(p, '_get_callback'):
+            urls.append(base + p.regex.pattern)
+        elif (isinstance(p, RegexURLResolver) or
+              hasattr(p, 'url_patterns') or
+              hasattr(p, '_get_url_patterns')):
+            try:
+                patterns = p.url_patterns
+            except ImportError:
```

Comment:
I'm not sure — the code this is based on catches an `ImportError` there. Removing it doesn't throw an `ImportError` with cfgov-refresh or any of its optional apps. Removing it.